ui/apps/platform/src/services/ComplianceCommon.ts

```diff
@@ -29,11 +28,25 @@ export type ComplianceCheckStatusCount = {
     status: ComplianceCheckStatus;
 };
 
+export type ComplianceBenchmark = {
+    name: string;
+    version: string;
+    description: string;
+    provider: string;
+    shortName: string;
```

Comment:
The comment from corresponding backend type is worthwhile to consider here.
https://github.com/stackrox/stackrox/blob/master/proto/api/v2/compliance_common.proto#L74-L75